src/main/java/com/teamabnormals/abnormals_core/core/mixin/client/SignTileEntityRendererMixin.java

```diff
@@ -0,0 +1,62 @@
+package com.teamabnormals.abnormals_core.core.mixin.client;
+
+import com.mojang.blaze3d.matrix.MatrixStack;
+import com.mojang.blaze3d.vertex.IVertexBuilder;
+import com.teamabnormals.abnormals_core.common.blocks.sign.ITexturedSign;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.WoodType;
+import net.minecraft.client.renderer.Atlases;
+import net.minecraft.client.renderer.IRenderTypeBuffer;
+import net.minecraft.client.renderer.RenderType;
+import net.minecraft.client.renderer.model.RenderMaterial;
+import net.minecraft.client.renderer.tileentity.SignTileEntityRenderer;
+import net.minecraft.tileentity.SignTileEntity;
+import net.minecraft.util.ResourceLocation;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.Redirect;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.util.function.Function;
+
+@Mixin(SignTileEntityRenderer.class)
+public abstract class SignTileEntityRendererMixin {
+
+	@Shadow @Final private SignTileEntityRenderer.SignModel model;
```

Comment:
Same as the other mixin critique, this annotation style on fields is inconsistent with other mixins.